modules/local/ctatsplicing/prepgenomelib/main.nf

```diff
@@ -0,0 +1,44 @@
+process CTATSPLICING_PREPGENOMELIB {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://data.broadinstitute.org/Trinity/CTAT_SINGULARITY/CTAT-SPLICING/ctat_splicing.v0.0.2.simg' :
+        'docker.io/trinityctat/ctat_splicing:0.0.2' }"
```

Comment:
Sorry just a little comment I realised. It seems this tool won't run with conda profile. Could you please raise an error in `utils_nfcore_rnafusion_pipeline` if users attempt to run rnafusion with `--profile conda`? This is to warn and terminate the pipeline early.

Reply:
Thank you! I added the message :)